R/sf.R

```diff
@@ -622,23 +746,85 @@ sf_rescale01_x <- function(x, range) {
 }
 
 
-#' @param crs Use this to select a specific CRS. If not specified, will
-#'   use the CRS defined in the first layer.
+#' @param crs Use this to select a specific coordinate reference system (CRS).
+#'   If not specified, will use the CRS defined in the first layer.
 #' @param datum CRS that provides datum to use when generating graticules
+#' @param label_axes Character vector or named list of character values
+#'   specifying which graticule lines (meridians or parallels) should be labeled on
+#'   which side of the plot. Meridians are indicated by `"E"` (for East) and
+#'   parallels by `"N"` (for North). Default is `"--EN"`, which specifies
+#'   (clockwise from the top) no labels on the top, none on the right, meridians
+#'   on the bottom, and parallels on the left. Alternatively, this setting could have been
+#'   specified with `list(left = "N", bottom = "E")`.
+#'
+#'   This parameter can be used alone or in combination with `label_graticule`.
+#' @param label_graticule Character vector indicating which graticule lines should be labeled
+#'   where. Meridians run north-south, and the letters `"N"` and `"S"` indicate that
+#'   they should be labeled on their north or south end points, respectively.
+#'   Parallels run east-west, and the letters `"E"` and `"W"` indicate that they
+#'   should be labeled on their east or west end points, respectively. Thus,
+#'   `label_graticule = "SW"` would label meridians at their south end and parallels at
+#'   their west end, whereas `label_graticule = "EW"` would label parallels at both
+#'   ends and meridians not at all. Because meridians and parallels can in general
```

Comment:
I want to make sure I understand this last sentence correctly (and, if I don't, perhaps some clarification is in order). `label_graticule` is oriented to the map, not to the plot panel. So, `label_graticule = "SW"`, for example, is not guaranteed to be in the bottom left corner of the plot panel. Is that kinda right?

Reply:
Yes, you understand correctly. In my experience, the result from `label_graticule` specifications can look unexpected for rotated coordinate systems, and it usually takes me a minute or so to verify that it's actually correct. See example.

``` r
library(sf)
#> Linking to GEOS 3.6.1, GDAL 2.1.3, proj.4 4.9.3
library(ggplot2) 

nc <- st_read(system.file("gpkg/nc.gpkg", package="sf"), quiet = TRUE)
ggplot() + geom_sf(aes(fill = AREA), data=nc) + 
  coord_sf(crs = 3338, label_graticule = "SW")
```

![](https://i.imgur.com/nJyeAV2.png)

Created on 2018-09-15 by the [reprex package](http://reprex.tidyverse.org) (v0.2.0).

Reply:
Great. Yeah, it makes sense, I just wanted to double check.